src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2123,10 +2126,13 @@ bool registrationsQuery
 
     MimeType                  mimeType = JSON;
     std::vector<BSONElement>  queryContextRegistrationV = getFieldF(r, REG_CONTEXT_REGISTRATION).Array();
+    std::string               format                    = getStringFieldF(r, REG_FORMAT);
+    ProviderFormat            providerFormat            = (format == "")? PfJson : (format == "JSON")? PfJson : PfV2;  // FIXME PR: Make V2 default format
```

Comment:
Default format should be V1 to keep backward compatibility behevaiour in registrations created before providerFormat was invended. I understand current code is correct, the removal of the FIXME PR mark is pending.

Reply:
Hmm, did you not say default format should be V2 just a while ago?
Misunderstanding, let's try to find out what has happened

Reply:
I'm probably thinking of a missing `legacyForwarding` in the creation of a registration.
There we said that the default should be V2.
 It's a bit strange but as the same time it makes sense. Regs created before legv1acyForwarding should probably be NGSIv1.

Reply:
Fixed in 2ca5e661c